app/assets/javascripts/components/locator-map.js

```diff
@@ -85,7 +85,7 @@
 
       this.map.setView([latitude, longitude], 8);
 
-      L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+      L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
```

Comment:
Found the solution in [this Leaflet issue](https://github.com/Leaflet/Leaflet/issues/3186).